### ISSUE TITLE
Hardware Requirments

### ISSUE THREAD

**Issue by TalDerei** [0xPolygonHermez/zkevm-prover#19]
I need to set up the prover in order to set up the zkevm-node. Or is there a way to setup the zkevm-node without setting up the prover?

Also, I have the cloud compute resources to run the prover. Can you tell me the rough vCPUs/memory/disk required on the VM so the prover doesn't fail to run. 

Thanks!

**Comment by jbaylina** [comment]
We are setting up a testnet and fixing some bugs.  We will let you know when it's ready.

**Comment by TalDerei** [comment]
Right, I'm doing some internal testing myself. Can you help me answer the questions above, because I can't find the answer to those in the docs? Thanks @jbaylina!

**Comment by krakhit** [comment]
Any updates on the hardware requirements?

**Comment by ItsFunny** [comment]
Any updates?

**Comment by arnaubennassar** [comment]
About hardware requirements:

- Executor (needed to run a synchronizer and JSON RPC): 16G RAM 4 cores
- Prover (only needed to run an aggregator): 1TB RAM, 128 cores

I believe there have been improvements on the prover requirements, but at least a few weeks ago we were using machines with this specs.

Note that in order to have a node that synchronizes and expose the JSON RPC interface it should be enough with 16G RAM and 4 cores, more info [here](https://github.com/0xPolygonHermez/zkevm-node/blob/develop/docs/production-setup.md)

> Or is there a way to setup the zkevm-node without setting up the prover first?

Yes the node is modular and there is no need to run the aggregator component (which is the only one that needs the high hardware requirements)

**Comment by Hodgeson** [comment]
Any updates?
